core/commands/stat.go

```diff
@@ -33,10 +33,10 @@ var statBwCmd = &cmds.Command{
 		ShortDescription: ``,
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("peer", "p", "Specify a peer to print bandwidth for."),
-		cmds.StringOption("proto", "t", "Specify a protocol to print bandwidth for."),
-		cmds.BoolOption("poll", "Print bandwidth at an interval. Default: false."),
-		cmds.StringOption("interval", "i", "Time interval to wait between updating output."),
+		cmds.StringOption("peer", "p", "Specify a peer to print bandwidth for.").Default("overall bandwidth"),
+		cmds.StringOption("proto", "t", "Specify a protocol to print bandwidth for.").Default("all protocols"),
```

Comment:
neither of these values are 'valid options'. If i run `ipfs stats bw --proto="all protocols"` its not going to work.

Setting the default option value here is going to require a bit more in-depth logic changes in the code. We will probably need to allow the user to specify `"all"`